### PR TITLE
AITOOL 6017 - Reduce high vulnerabilities

### PR DESCRIPTION
**Pull request by joseangel-mm**
# Jira ticket: [AITOOL-6017](https://lampkicking.atlassian.net/browse/AITOOL-6017)

## Purpose
Reduce as many as possible `high` vulnerabilities.

This is the current status after some updates:
Screenshot 2024-09-27 at 08 45 59

There are some high vulnerabilities that cannot be removed:
* `axios  1.3.2 - 1.7.3` -> It comes from the `face-api.js` package used in the FCM.
* `node-fetch  <=2.6.6` -> It comes from the `face-api.js` package used in the FCM.
* `nth-check  <2.0.1` -> It comes from the `react-scripts` package. There is no fix in the latest versions.
* `postcss  <8.4.31` -> It comes from the `react-scripts` package. There is no fix in the latest versions.
* `rollup  <3.29.5` -> It comes from the `react-scripts` package. There is no fix in the latest versions.

[AITOOL-6017]: https://lampkicking.atlassian.net/browse/AITOOL-6017?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "axios": "1.6.8",
     "clsx": "^1.1.1",
     "dotenv-flow": "^3.2.0",
-    "express": "^4.17.2",
+    "express": "4.21.0",
```
Comment (pegondo) [inline package.json:15]: Do we mind the minor?:
```suggestion
    "express": "^4.21.0",
```

**Review by pegondo — COMMENTED** [review]
Just one question

File: package.json (hunk 1 above)
Comment (joseangel-mm) [reply package.json]: Yes, that's a good point. Already added `^`.

**Review by DionisioRD — APPROVED** [review]

**Review by MiguelJS — APPROVED** [review]

**Review by pegondo — APPROVED** [review]
LGTM, thanks.

**Pull request merged**